lib/1.2/dml-builtins.dml

```diff
@@ -386,6 +383,13 @@ template bank {
     is _hard_reset;
     is _soft_reset;
 
+    // Redundant due to SIMICS-19009; banks must be top-level in 1.2 devices
+    //
+    // if ($parent.objtype == "group" ? $parent._simics_namespace_clash
+    //     : false) {
+    //     error "Cannot declare a bank underneath a group named 'port'";
+    // }
```

Comment:
I see why you write this, but commented-out code is a pretty strong anti-pattern. Can it be expressed concisely enough on some other form, e.g. English?

Reply:
Yeah, I can.